pkg/providers/vsphere/config/template-cp.yaml

```diff
@@ -176,6 +176,16 @@ spec:
           name: audit-log
           pathType: FileOrCreate
           readOnly: false
+{{- if .awsIamAuth}}
```

Comment:
just to confirm, all the template changes are same for ubuntu and BR?

Reply:
Yes. Since I had to mount certain volumes to the authenticator server which is outside the cloud init template, I kept the paths for all providers how BR would like it :)
There were some discussions to maybe do it differently later.